Kitodo/src/test/java/org/kitodo/production/interfaces/activemq/KitodoScriptProcessorIT.java

```diff
@@ -0,0 +1,440 @@
+                return 0;
+            }
+
+            @Override
+            public String getString(String name) throws JMSException {
+                return null;
+            }
+
+            @Override
+            public byte[] getBytes(String name) throws JMSException {
+                return null;
+            }
+
+            @Override
+            public Object getObject(String name) throws JMSException {
+                return null;
+            }
+
+            @Override
+            public Enumeration getMapNames() throws JMSException {
+                return null;
+            }
+
+            @Override
+            public void setBoolean(String name, boolean value) throws JMSException {
+
+            }
+
+            @Override
+            public void setByte(String name, byte value) throws JMSException {
+
+            }
+
+            @Override
+            public void setShort(String name, short value) throws JMSException {
+
+            }
+
+            @Override
+            public void setChar(String name, char value) throws JMSException {
+
+            }
+
+            @Override
+            public void setInt(String name, int value) throws JMSException {
+
+            }
+
+            @Override
+            public void setLong(String name, long value) throws JMSException {
+
+            }
+
+            @Override
+            public void setFloat(String name, float value) throws JMSException {
+
+            }
+
+            @Override
+            public void setDouble(String name, double value) throws JMSException {
+
+            }
+
+            @Override
+            public void setString(String name, String value) throws JMSException {
+
+            }
+
+            @Override
+            public void setBytes(String name, byte[] value) throws JMSException {
+
+            }
+
+            @Override
+            public void setBytes(String name, byte[] value, int offset, int length) throws JMSException {
+
+            }
+
+            @Override
+            public void setObject(String name, Object value) throws JMSException {
+
+            }
+
+            @Override
+            public boolean itemExists(String name) throws JMSException {
+                return false;
+            }
+        }) {
+            @Override
+            public String getMandatoryString(String s) {
+                return "action:test";
+            }
+
+            @Override
+            public Collection<Integer> getCollectionOfInteger(String s) {
+                return Collections.singletonList(1);
+            }
+        };
+
+        // the object to be tested
+        KitodoScriptProcessor underTest = new KitodoScriptProcessor();
+
+        // organize return of results
+        List<String> scriptResult = new ArrayList<>();
+        List<Process> processesResult = new ArrayList<>();
+        Field serviceField = KitodoScriptProcessor.class.getDeclaredField("kitodoScriptService");
```

Comment:
Is using the Mockito framework may a better / cleaner solution than using field injection here?

Reply:
I don't know exactly what that means. I've never used it before. A quick look, it might be helpful, maybe. Unfortunately, I don't have the time to learn this right now, so maybe at some point, I can do a change pull request again. Or, can you suggest what the code should look like?

Reply:
I will look into this and if I get no right solution in the next time your way here is okay. It can be changed in the future too.